core/templates/pages/contributor-dashboard-page/custom-snackbar/undo-snackbar.component.html

```diff
@@ -0,0 +1,35 @@
+<div class="snackbar-container">
```

Comment:
I am not sure whether this snackbar component should be inside contributor dashboard page or core/components directory. @seanlip WDYT?

Reply:
I would put it in core/components since it has nothing to do with the contributor dashboard page intrinsically. Good catch!

Reply:
@Ash-2k3 You didn't respond to this comment.

Reply:
Done, Thanks!